src/templates/container-hotkey.jsx

```diff
@@ -1,53 +1,53 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
```

Comment:
There's a codemod ready for this: https://github.com/reactjs/react-codemod#react-proptypes-to-prop-types
Maybe it makes sense to just do it whenever we update to 15.5.